spec/facades/books/show_facade_spec.rb

```diff
@@ -6,16 +6,24 @@
 RSpec.describe ::Books::ShowFacade do
   context 'book rating' do
     before(:each) do
-      @book = create(:book)
+      @book = create(:book).decorate
```

Comment:
`before { create(:book).decorate }`

if you need book variable in your tests use `let`:
```
let(:book) { create(:book) }
```